packages/core/src/internal/interfaces/index.ts

```diff
@@ -16,6 +16,10 @@ export type StoplightStatusTypes = 'success' | 'warning' | 'danger';
 
 export type StatusTypes = 'info' | StoplightStatusTypes;
 
+export type AlertStatusTypes = 'info' | 'success' | 'warning' | 'danger' | 'unknown' | 'loading';
+
+export type AlertGroupTypes = 'default' | 'banner' | 'light';
```

Comment:
nit: component specific interfaces and code should probably stay within its directory

Reply:
they were put here early on because we initially thought we'd have alerts and app-alerts as separate components. they were intended to be shared. i'll relocate them.